### DISCUSSION TITLE
File naming template

### DISCUSSION THREAD

**Discussion by ldiegos** [FormerLurker/Octolapse#681] [Ideas]
Hi, good morning,

What do you think about making global to all the render options the filename template?
{FAILEDFLAG}{FAILEDSEPARATOR}{GCODEFILENAME}_{PRINTENDTIME}

It seems, that you have to go one by one, changing the filename template and a message appear that the template will no longer be update, because you have modified it.

For example adding the "fps" to the name, i have to go template by template adding it. 
example: {FAILEDFLAG}{FAILEDSEPARATOR}{GCODEFILENAME}_{PRINTENDTIME}_{FPS}fps

Kind Regards,
Luis.

**Comment by FormerLurker** [comment]
Sorry, I'm not sure what your request is exactly.  Are you asking for changes in one profile to affect all profiles for the rendering filename template?  I can see how that would make it easier to change all of the profiles, but also I can see that upsetting other users who want the opposite behavior.  Let me know first if I'm correct about your request.

**Comment by ldiegos** [comment]
The idea is to put the file name outside the profiles.

I don't see the issue with getting upset, if you have several profiles and you want to add the fps to identify the profile in the video, just change it in one site and that it. After that all the videos will identify the render profile with the fps in the name.

But as you told, maybe we the users ;) want to change the same thing in all the profiles lol.

i will try to get use to change all the profiles to adapt it to my way of thinking with the video file name format. At the end, i don't think that i will change the video name so often.

Thank you so much for your time
Luis.

P.S: why this section of ideas is completely empty? nobody have ideas to improve octolapse?

**Comment by FormerLurker** [comment]
> The idea is to put the file name outside the profiles.

Ok, that makes sense.  So this would reside maybe in the main settings?

If I do this the hardest part will be to figure out exaclty HOW to migrate to the new way.  I guess I could add some kind of override in the main setting for the file name template (say use per-profile naming or something like that with a checkbox), leave the rendering template alone.  Then, for ppl who are migrating, I could 'tick' that checkbox, and for new installs it could be unchecked.

Does that make sense (sorry, it's early and I've just now gotten my coffee)?

> why this section of ideas is completely empty? nobody have ideas to improve octolapse?

The discussions are a relatively new feature, and I just turned them on :)

**Comment by ldiegos** [comment]
It sounds great!!!!, lets wait for other mates and known their opinion.
